modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarRight.tsx

```diff
@@ -91,32 +93,35 @@ const ManagementToolbarRight: React.FC<ManagementToolbarRightProps> = ({
 			)}
 
 			{display.columns && (
-				<ClayDropDownWithItems
-					items={columns}
+				<ClayPopover
+					alignPosition="bottom-right"
+					className="body-columns popover-management-toolbar"
```

Comment:
Those classnames are too vague

```suggestion
					className="body-columns popover-management-toolbar"
```